devel/oxenmq/Portfile

```diff
@@ -47,5 +47,4 @@ if {[string match *gcc* ${configure.compiler}]} {
 }
 
 test.run                yes
-test.cmd                ${cmake.build_dir}/tests/tests
-test.target
+test.target             tests
```

Comment:
@TheRealKeto With this change tests are not _run,_ only _built_. Please change it back or fix it otherwise.

```
--->  Testing oxenmq
Executing:  cd "/opt/local/var/macports/build/_opt_local_var_macports_sources_rsync.macports.org_macports_release_tarballs_ports_devel_oxenmq/oxenmq/work/build" && /usr/bin/make tests 
/opt/local/bin/cmake -S/opt/local/var/macports/build/_opt_local_var_macports_sources_rsync.macports.org_macports_release_tarballs_ports_devel_oxenmq/oxenmq/work/oxen-mq-1.2.15 -B/opt/local/var/macports/build/_opt_local_var_macports_sources_rsync.macports.org_macports_release_tarballs_ports_devel_oxenmq/oxenmq/work/build --check-build-system CMakeFiles/Makefile.cmake 0
/usr/bin/make  -f CMakeFiles/Makefile2 tests
/opt/local/bin/cmake -S/opt/local/var/macports/build/_opt_local_var_macports_sources_rsync.macports.org_macports_release_tarballs_ports_devel_oxenmq/oxenmq/work/oxen-mq-1.2.15 -B/opt/local/var/macports/build/_opt_local_var_macports_sources_rsync.macports.org_macports_release_tarballs_ports_devel_oxenmq/oxenmq/work/build --check-build-system CMakeFiles/Makefile.cmake 0
/opt/local/bin/cmake -E cmake_progress_start /opt/local/var/macports/build/_opt_local_var_macports_sources_rsync.macports.org_macports_release_tarballs_ports_devel_oxenmq/oxenmq/work/build/CMakeFiles 21
/usr/bin/make  -f CMakeFiles/Makefile2 tests/CMakeFiles/tests.dir/all
/usr/bin/make  -f CMakeFiles/oxenmq.dir/build.make CMakeFiles/oxenmq.dir/depend
cd /opt/local/var/macports/build/_opt_local_var_macports_sources_rsync.macports.org_macports_release_tarballs_ports_devel_oxenmq/oxenmq/work/build && /opt/local/bin/cmake -E cmake_depends "Unix Makefiles" /opt/local/var/macports/build/_opt_local_var_macports_sources_rsync.macports.org_macports_release_tarballs_ports_devel_oxenmq/oxenmq/work/oxen-mq-1.2.15 /opt/local/var/macports/build/_opt_local_var_macports_sources_rsync.macports.org_macports_release_tarballs_ports_devel_oxenmq/oxenmq/work/oxen-mq-1.2.15 /opt/local/var/macports/build/_opt_local_var_macports_sources_rsync.macports.org_macports_release_tarballs_ports_devel_oxenmq/oxenmq/work/build /opt/local/var/macports/build/_opt_local_var_macports_sources_rsync.macports.org_macports_release_tarballs_ports_devel_oxenmq/oxenmq/work/build /opt/local/var/macports/build/_opt_local_var_macports_sources_rsync.macports.org_macports_release_tarballs_ports_devel_oxenmq/oxenmq/work/build/CMakeFiles/oxenmq.dir/DependInfo.cmake "--color="
/usr/bin/make  -f CMakeFiles/oxenmq.dir/build.make CMakeFiles/oxenmq.dir/build
make[3]: Nothing to be done for `CMakeFiles/oxenmq.dir/build'.
[ 38%] Built target oxenmq
/usr/bin/make  -f tests/CMakeFiles/tests.dir/build.make tests/CMakeFiles/tests.dir/depend
cd /opt/local/var/macports/build/_opt_local_var_macports_sources_rsync.macports.org_macports_release_tarballs_ports_devel_oxenmq/oxenmq/work/build && /opt/local/bin/cmake -E cmake_depends "Unix Makefiles" /opt/local/var/macports/build/_opt_local_var_macports_sources_rsync.macports.org_macports_release_tarballs_ports_devel_oxenmq/oxenmq/work/oxen-mq-1.2.15 /opt/local/var/macports/build/_opt_local_var_macports_sources_rsync.macports.org_macports_release_tarballs_ports_devel_oxenmq/oxenmq/work/oxen-mq-1.2.15/tests /opt/local/var/macports/build/_opt_local_var_macports_sources_rsync.macports.org_macports_release_tarballs_ports_devel_oxenmq/oxenmq/work/build /opt/local/var/macports/build/_opt_local_var_macports_sources_rsync.macports.org_macports_release_tarballs_ports_devel_oxenmq/oxenmq/work/build/tests /opt/local/var/macports/build/_opt_local_var_macports_sources_rsync.macports.org_macports_release_tarballs_ports_devel_oxenmq/oxenmq/work/build/tests/CMakeFiles/tests.dir/DependInfo.cmake "--color="
/usr/bin/make  -f tests/CMakeFiles/tests.dir/build.make tests/CMakeFiles/tests.dir/build
make[3]: Nothing to be done for `tests/CMakeFiles/tests.dir/build'.
[100%] Built target tests
/opt/local/bin/cmake -E cmake_progress_start /opt/local/var/macports/build/_opt_local_var_macports_sources_rsync.macports.org_macports_release_tarballs_ports_devel_oxenmq/oxenmq/work/build/CMakeFiles 0
```
(Or do I miss something?)

Reply:
I believe running `tests` is broken, since the binary expects `oxenmq` to be installed. `tests` uses references of dylibs from /opt/local/lib, rather than their local references (inside `cmake.build_dir`). I reverted the change, but I did make the mistake of simply building `tests`.

Reply:
@TheRealKeto Thank you, I will check that. I believe running tests worked for me when I made the port, but maybe I set environment manually and forgot about it or had `oxenc` installed and didn’t notice the issue. This should be fixable by passing DYLIB_LIBRARY_PATH to test environment. I gonna deal with that in a separate PR anyway, since there are two other oxen* ports which use similar testing arrangement.